script.js

```diff
@@ -0,0 +1,90 @@
+window.onload = start;
+var boxes = document.getElementsByTagName("td");
+var turnText = document.querySelector(".playerTurn");
+var counter = 1;
+var winCounter = 0;
+var OMoves = [];
+var XMoves = [];
+
+//easiest way to determine how someone wins.
+var winningCombinations = [[0,1,2],[3,4,5],[6,7,8],
+[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]];
+
+function start(){
+  addXandOListener();
+  addResetListener();
+}
+
+function addXandOListener(){
+  for (var i = boxes.length - 1; i >= 0; i--) {
+    boxes[i].addEventListener("click", addXorO);
+  }
+}
+
+function addXorO(event){
+  if (event.target.innerHTML.length === 0){
```

Comment:
You should be able to use "this" on the callback function. So instead of `event.target.innerHTML`, you could do `this.innerHTML`